src/components/balances/TokenListSelect/index.tsx

```diff
@@ -26,6 +28,13 @@ const TokenListSelect = () => {
     dispatch(setTokenList(selectedString))
   }
 
+  const handleTrack = (label: 'Open' | 'Close') => {
```

Comment:
Could we make these labels more specific? Otherwise there might be some overlaps in GA. Maybe something like "Open token list", "Close token list"

Reply:
I've changed it.